README.md

```diff
@@ -1,73 +1,201 @@
+    mapping of actors (subjects) to resources (object) to operations
+    (predicate). For instance, a CMR ACL might specify that all Registered
+    users have READ access to ASTER data or all users in a provider operations
+    group have permissions to ingest data for a particular provider.
   - API Docs: https://cmr.earthdata.nasa.gov/access-control/site/access_control_api_docs.html
 
 ## Our Development Environment
-  - Mac OSX
-  - Atom: https://atom.io/
-  - Proto-Repl: https://atom.io/packages/proto-repl
-    - Installed and configured according to this guide: https://git.io/atom_clojure_setup
+
+- Mac OSX
+- Atom: https://atom.io/
+- Proto-Repl: https://atom.io/packages/proto-repl
+  - Installed and configured according to this guide: https://git.io/atom_clojure_setup
 
 ## Prerequisites
+
 - Java 1.8.0 or higher
 - Leiningen (http://leiningen.org) 2.5.1 or above.
-  - We've had success with Homebrew and with the install script on the Leiningen website.
+  - We've had success with Homebrew and with the install script on the
+    Leiningen website.
 
 ## Building and Running the CMR
-The CMR is a system consisting of multiple services. The services can be run individually or can run in a single process. Running in a single process makes local development easier because it avoids having to start many different processes. The dev-system project allows the CMR to be run from a single REPL or Jar file. If you're developing a client against the CMR you can build and run the entire CMR with no external dependencies from this Jar file and use that instance for local testing. The sections below contain instructions for running the CMR as a single process or as multiple processes.
+
+The CMR is a system consisting of multiple services. The services can be run
+individually or can run in a single process. Running in a single process makes
+local development easier because it avoids having to start many different
+processes. The dev-system project allows the CMR to be run from a single REPL
+or Jar file. If you're developing a client against the CMR you can build and
+run the entire CMR with no external dependencies from this Jar file and use
+that instance for local testing. The sections below contain instructions for
+running the CMR as a single process or as multiple processes.
 
 #### Building and Running CMR Dev System in a REPL
 
-1. Install Oracle JDBC Jars into your local maven repository following instructions in `oracle-lib/README.md`. The CMR must have these libraries to build but it does not depend on Oracle DB when running locally. It uses a local in memory database by default.
-2. `cd cmr/dev-system`
-3. `./support/setup_local_dev.sh`
+1. Install Oracle JDBC Jars into your local maven repository following
+   instructions in `oracle-lib/README.md`. The CMR must have these libraries to
+   build but it does not depend on Oracle DB when running locally. It uses a
+   local in memory database by default.
+2. `cd cmr`
```

Comment:
Since the repo name changed it should be:

`cd cmr` -> `cd Common-Metadata-Repository`